resources/migrations/permissions/mariadb_enforce_no_data_permissions_on_update.sql

```diff
@@ -0,0 +1,11 @@
+CREATE TRIGGER enforce_no_data_permissions_on_update
```

Comment:
This doesn't hurt to have, of course, but I don't think we ever actually update `permissions` rows. All perms writes are are deletes and then inserts. So this trigger could probably be omitted with no issue.

Reply:
Good point - I think I will omit this then, as it requires a separate changeset to create and makes things more confusing.